docs/sce_file_contents.md

```diff
@@ -0,0 +1,131 @@
+# Single-cell gene expression file contents
+
+Single cell or single-nuclei gene expression data (filtered or unfiltered) is provided for use with R as an RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
+This object contains the expression data, cell and gene metrics, associated metadata, and, in the case of multimodal data like CITE-seq, data from additional cell-based assays.
+
+We highly encourage you to familiarize yourself with the general object structure and functions available as part of the [`SingleCellExperiment` package](https://bioconductor.org/packages/3.13/bioc/html/SingleCellExperiment.html) from Bioconductor.
+Below we present some details about the specific contents of the objects we provide.
+
+To begin, you will need to load the `SingleCellExperiment` package and read the RDS file:
+
+```r
+library(SingleCellExperiment)
+sce <- readRDS("SCPCL000000_filtered.rds")
+```
+
+## Common fields
+
+### Expression counts
+
+For both unfiltered and filtered single-cell and single-nucleotide experiments, the primary cell by gene RNA-seq expression count data is provided as the `"counts"` assay.
+Each column represents a cell or droplet, each row a gene.
+Column names are cell barcode sequences and row names are Ensembl gene IDs. 
+This data is stored as a sparse matrix that can be accessed with the following R code:
```

Comment:
```suggestion
The data is stored as a sparse matrix, and each column represents a cell or droplet, each row a gene.
Column names are cell barcode sequences and row names are Ensembl gene IDs. 
The `counts` assay can be accessed with the following R code:
```
I think we should first say this is a matrix and then describe the contents of the matrix.